packages/augur-ui/src/modules/auth/actions/login-with-portis.ts

```diff
@@ -0,0 +1,76 @@
+import { updateSdk } from "modules/auth/actions/update-sdk";
+import { toChecksumAddress } from "ethereumjs-util";
+import { ThunkDispatch } from "redux-thunk";
+import { Action } from "redux";
+import { NodeStyleCallback } from "modules/types";
+import { Web3Provider } from "ethers/providers";
+import Portis, { INetwork } from "@portis/web3";
+import Web3 from "web3";
+import { updateIsLoggedAndLoadAccountData } from "modules/auth/actions/update-is-logged-and-load-account-data";
+import { ACCOUNT_TYPES } from "modules/common/constants";
+import { AppState } from "store";
+
+// TODO find home for all wallet API keys
+const PORTIS_API_KEY = "b67817cf-8dd0-4116-a0cf-657820ddc019";
+
+const getPortisNetwork = (networkId): false | string | INetwork  => {
+  const myPrivateEthereumNode = {
+    nodeUrl: "http://localhost:8545",
+    chainId: "104",
```

Comment:
we actually have many private node chainIds. See `augur/packages/augur-artifacts/src/addresses.json`
don't use anything with augur-node. use `getNetworkId()` in contractCalls.ts

Reply:
ahh that makes more sense!